src/main/java/com/googleinterns/zoomtube/transcriptParser/TranscriptParser.java

```diff
@@ -164,4 +164,17 @@ private Entity createTranscriptLineFromElement(Key lectureKey, Element transcrip
     return TranscriptLineUtil.createEntity(
         lectureKey, lineContent, lineStartMs, lineDurationMs, lineEndMs);
   }
+
+  /**
+   * Cleans the {@code content} of a transcript line by unescaping XML characters, and
+   * removing newlines.
+   */
+  private String cleanupTranscriptContent(String content) {
+    String unescapedXml = StringEscapeUtils.unescapeXml(content);
```

Comment:
can you add transcript line to the name of this

Reply:
not sure what you meant by this... I changed the names a little tho, tell me how it looks now

Reply:
`unescapedTranscriptLine`

Reply:
Ah, will do

Reply:
I changed it to `unescapedContent` b/c it's the content, not the line as a whole.  do you think that's ok?

Reply:
sure that'll work